lib/js/jscore.nim

```diff
@@ -110,3 +110,15 @@ proc parse*(l: JsonLib, s: cstring): JsRoot {.importcpp.}
 since (1, 5):
   func debugger*() {.importjs: "debugger@".}
     ## https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/debugger
+
+  template jsexport*(symbol: untyped; alias = "") =
+    ## https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/export
+    runnableExamples("-b:js -r:off"):
```

Comment:
`-b` flag is propagated from `nim doc -b:js` command so shouldn't be needed (please double check)